db/db.go

```diff
@@ -479,6 +479,18 @@ func GetAllEpochs() ([]uint64, error) {
 	return epochs, nil
 }
 
+// Count finalized epochs in range
```

Comment:
**Nitpick:** Could state that both `start` and `end` are inclusive (at least `end` is not always inclusive in "similar" functions).

Reply:
added to comment